### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by ypascal**
Hi Shammah,

Here is a pull request for internationalization support.
- Everything related to the setup of i18n has been moved to src/i18n/index.ts
- French translation is included

### CODE REVIEW

**Comment by schancel** [comment]
Wow, thank you so much for this. Do you have a Lotus address from Vase or ABC Pay you can send me?

File: src/pages/Settings.vue [hunk 1]
```diff
@@ -112,5 +134,16 @@ export default defineComponent({
   mounted() {
     this.contactRefreshInterval?.focus()
   },
+  watch: {
+    lang(lang) {
```
Comment (schancel) [inline src/pages/Settings.vue:138]: You can do this in the `setup` function, but this is okay. I'm surprised this isn't already a ref.  There's a `watch()` function. This is preferred: https://vuejs.org/guide/essentials/watchers.html#basic-example

**Review by schancel — APPROVED** [review]

**Comment by schancel** [comment]
One thing that would be amazingly helpful as well is the i18n linter. I don't quite understand how to set it up, but there's a lot of strings and whatnot that are out of sync with the i18n files.

File: src/pages/Settings.vue (hunk 1 above)
Comment (ypascal) [reply src/pages/Settings.vue:138]: I will look to improve that ; I'm not very familiar with vuejs

**Pull request merged**